web/src/features/landingPage/components/ToolCard.tsx

```diff
@@ -75,13 +75,13 @@ const ToolCard: React.FunctionComponent<ToolCardProps> = (props: ToolCardProps)
     const path = props.route
     if (path.startsWith('http')) {
       return (
-        <a className={classes.link} href={path} rel="noreferrer" target="_blank">
+        <a className={classes.link} href={path} rel="noreferrer">
```

Comment:
Why this change? We lose apps opening in new tabs with it

Reply:
It's an acceptance criteria in the ticket. BC Gov's standards for accessibility is that links shouldn't open in new tabs

Reply:
Oof ok